### DISCUSSION TITLE
How can I use nb against a local git repository?

### DISCUSSION THREAD

**Discussion by boldandbusted** [xwmx/nb#257] [Q&A]
I would like to have nb use the local git repository its files are within. How do I configure that? Thanks! :)

**Comment by boldandbusted** [comment]
Ah, it seems like PEBKAC. I created a subdirectory in an *existing repo*, and then `nb init` in that directory. This initialized a *new* repository in that directory, ignoring the parent.

Maybe someday I can figure out how to better init a notebook that is only one part of the larger git repo, but I think this is more about me than nb! :)

**Comment by mandarvaze** [comment]
This is a valid use case.  I would also like to create subdirectory (local notebook as mentioned in the documentation)

But when I tried `nb` as well as `nb notebooks init <existing_folder>` I get asked questions about name/email (which I assume will set `user.name` and `user.email` for git)

But parent is already a git repo, so those values are already set. I just need to be able to use `nb` within that folder

@xwmx Please suggest if this is supported
